### ISSUE TITLE
Chapter 5 typo

### ISSUE THREAD

**Issue by xyang2013** [rasbt/reasoning-from-scratch#206]
"Using mismatched combinations, such as torch.softmax with torch.sum or torch.log_softmax with torch.prod, is technically possible but mathematically incorrect."

**Comment by rasbt** [comment]
Thanks! But hm, I can't see the issue. Could you explain what the typo is in 

> "Using mismatched combinations, such as torch.softmax with torch.sum or torch.log_softmax with torch.prod, is technically possible but mathematically incorrect."

For reference

The correct matches are

1) torch.softmax with torch.prod
2) torch.log_softmax with torch.sum

I am closing it in hope it answers the question. But please let me know in case I missed anything.

**Comment by xyang2013** [comment]
Sorry. I see what you mean. I thought you meant about replacing. I read it incorrectly.
